azure-pipelines.yml

```diff
@@ -170,6 +170,64 @@ stages:
               summaryFileLocation: '$(System.DefaultWorkingDirectory)/**/coverage.xml'
               reportDirectory: '$(System.DefaultWorkingDirectory)/**/htmlcov'
 
+      - job: no_sqlalchemy
+        condition: or(eq(variables.isScheduled, true), eq(variables.isMain, true), eq(variables.isManual, true))
+
+        services:
+          postgres: postgres
+
+        variables:
+          GE_pytest_opts: ''
+
+        strategy:
+          matrix:
+            Python38:
+              python.version: '3.8'
+              GE_pytest_pip_opts: '--requirement requirements-dev.txt --constraint constraints-dev.txt'
+              pip_uninstall_packages: 'sqlalchemy'
```

Comment:
Do we plan on testing more versions? If not, what are your thoughts on moving away from a matrix and just using the single version?

Reply:
I'm not sure - was leaving in case we want to also add 3.9. Let me think on it.